docs/superpowers/specs/2026-05-15-tracker-git-preflight-design.md

```
@@ -0,0 +1,316 @@
+  requires: [git]
+  defaults:
+    model: claude-opus-4-7
+```
+
+- The `requires:` list declares environmental dependencies the workflow expects to be satisfied at run start.
+- v0.29.0 implements **only `git`**. Unrecognized entries (`docker`, `gh`, `jq`, etc.) parse successfully and emit a single warning per entry: `tracker: requires "<dep>" is not yet implemented; ignoring`. This lets workflow authors forward-declare without breaking on older tracker versions.
+- The check runs once at the top of `tracker run`, before any node executes.
+
+### CLI flags
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)